src/sagemaker/experiments/run.py

```diff
@@ -843,7 +844,6 @@ def list_runs(
     created_after: Optional[datetime.datetime] = None,
     sagemaker_session: Optional["Session"] = None,
     max_results: Optional[int] = None,
-    next_token: Optional[str] = None,
```

Comment:
To remove a field from a public method can be a breaking change. Is it causing any issues so we need to remove it?

Or are we sure the `list_runs` is not widely used? If so it's fine to remove it

Reply:
the argument is unusable as it currently is, so no one is currently using it

Reply:
+1 on @qidewenwhen 's comment.

Can we keep this parameter in public method but instead doing nothing with it?

Reply:
ill separate the next_token change from the bug fix and we can discuss what to do with it separately